Yoast/javascript
# Change text results of Portuguese paper

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Changes the recommended maximum result of `portuguesePaper2` in `fullTextTests` folder and deletes a word that is also a preposition from participle list.
* [Yoast SEO Free] Improves the accuracy of passive voice detection in Portuguese.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your site to Portuguese;
* Add a lorem ipsum text of min 150 words;
* Add a sentence containing the word `do` : 
 `Os gatos são do.`
* Check that this sentence is not marked as passive.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-691
